packages/core/src/App/Containers/Modals/mt5-notification/mt5-notification-description.tsx

```diff
@@ -5,56 +5,39 @@ import { Localize } from '@deriv/translations';
-                    <ul>
-                        <li className='mt5-notification-list'>
-                            <Text as='p' size='xs'>
-                                <Localize
-                                    i18n_default_text='- iOS: Swipe left on the account and tap <0>Delete</0>.'
-                                    components={[<strong key={0} />]}
-                                />
-                            </Text>
-                        </li>
-                        <li className='mt5-notification-list'>
-                            <Text as='p' size='xs'>
-                                <Localize
-                                    i18n_default_text='- Android: Tap the account, open <0>Options</0>, and tap <0>Delete</0>.'
-                                    components={[<strong key={0} />]}
-                                />
-                            </Text>
-                        </li>
-                    </ul>
-                </li>
-                <li className='mt5-notification-list-container-item'>
+                <li className='mt5-notification-list'>
                     <Text as='p' size='xs'>
                         <Localize
-                            i18n_default_text='Search for the broker <0>Deriv Holdings (Guernsey) Limited</0> and select it.'
-                            components={[<strong key={0} />]}
+                            i18n_default_text='Log back in to MT5 after 7:30 GMT on 20 Oct 2023 if you’re having difficulty logging in to MT5 as we’re making some updates to our MT5 platform. <0>Follow these steps</0> to log back in to MT5.'
+                            components={[
+                                <a
+                                    target='_blank'
+                                    rel='noopener noreferrer'
+                                    href={
+                                        'https://community.deriv.com/t/log-in-using-mt5-pc-or-mobile-app-application-guideline/49622'
+                                    }
+                                    key={0}
+                                />,
+                            ]}
                         />
                     </Text>
                 </li>
-                <li className='mt5-notification-list-container-item'>
+                <li className='mt5-notification-list'>
                     <Text as='p' size='xs'>
-                        <Localize i18n_default_text='Re-add your MT5 account using the same log in credentials.' />
+                        <Localize
+                            i18n_default_text='If you need further assistance, let us know via <0>live chat</0>.'
+                            components={[
+                                <a
+                                    onClick={() => window.LC_API.open_chat_window()}
+                                    key={0}
+                                    style={{ color: '#551A8B', cursor: 'pointer', textDecoration: 'underline' }}
```

Comment:
Why do we need inline style?

Reply:
Its an anchor tag and we cant use the className inside this component. I tried the className but it doesn't work